src/GraphQL.Tests/Types/ObjectGraphTypeTests.cs

```diff
@@ -9,8 +9,16 @@ public class ObjectGraphTypeTests
     {
         private class TestInterface : InterfaceGraphType { }
 
-        [GraphQLMetadata(Name = ":::")]
-        private class TypeWithInvalidName : ObjectGraphType { }
+        private class TypeWithInvalidName : ObjectGraphType
```

Comment:
The only test in the entire project that utilized `[GraphQLMetadata]` on a graphtype class, which was only testing that invalid names cannot be set on a graph type.

Reply:
👍